lighthouse-core/audits/accessibility/html-lang-valid.js

```diff
@@ -37,9 +37,12 @@ class HTMLLangValid extends AxeAudit {
       title: str_(UIStrings.title),
       failureTitle: str_(UIStrings.failureTitle),
       description: str_(UIStrings.description),
-      requiredArtifacts: ['Accessibility'],
     };
   }
+
+  static get requiredArtifacts() {
```

Comment:
Could define this in the Axe audit base class, instead of in each axe audit. sg?